MODULE.bazel

```diff
@@ -9,7 +9,7 @@ module(
 bazel_dep(name = "aspect_bazel_lib", version = "1.30.2")
 
 # Note: only used for provider symbols, we don't spawn nodejs actions
-bazel_dep(name = "aspect_rules_js", version = "1.29.2")
+bazel_dep(name = "aspect_rules_js", version = "1.31.0")
```

Comment:
Minimum rules_js version for Bazel 7 support https://pro.aspect.build/guides/bzlmod.pro/#bazel-7-minimum-versions